WorldMoodTracker/js/components/WorldMoodTracker.js

```diff
@@ -1,12 +1,15 @@
 import React from 'react';
+import Header from './Header';
+import WorldMap from './WorldMap';
+import Footer from './Footer';
 
 export default class WorldMoodTracker extends React.Component {
     render() {
         return (
             <div>
-                <h1>
-                    World Mood Tracker
-                </h1>
+                <Header/>
+                <WorldMap/>
+                <Footer />
             </div>
```

Comment:
my bad,  overlooked the line where you have mentioned the styles.
Just a doubt, why providing footer only on small screens, is it only because on large screen users dont have to scroll down but on xs screens users might have to scroll down or there is any other reason?